README.md

```diff
@@ -78,7 +78,7 @@ Possible options are listed in the following table. More details are [in the cod
 |------------|-----------|-------------------|----------|
 | `position` | `string`  | Position of the control | `topleft` |
 | `layer` | [`ILayer`](http://leafletjs.com/reference.html#ilayer)  | The layer that the user's location should be drawn on. | a new layer |
-| `setView` | `boolean`  or `string`  | Set the map view (zoom and pan) to the user's location as it updates. Options are `false`, `'once'`, `'always'`, `and 'untilPan'` | `untilPan` |
+| `setView` | `boolean`  or `string`  | Set the map view (zoom and pan) to the user's location as it updates. Options are `false`, `'once'`, `'always'`, `'untilPan'`, or [`'untilPanOrZoom'`](#213) | `untilPan` |
```

Comment:
Hmm, we don't link to PRs for any other property so let's not do it here either.

Reply:
it's not the PR.  it's the "vote"

Reply:
the idea is to make the discussion discoverable.  otherwise i imagine people who do want to use it won't know it's available for discussion